dockerflow.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+
+import flask
+
+from git_hg_sync import PID_FILEPATH
+
+app = flask.Flask(__name__)
+
+
+@app.route("/__lbheartbeat__")
+def lb_heartbeat() -> flask.Response:
+    return flask.Response("ok")
+
+
+@app.route("/__heartbeat__")
+def heartbeat() -> flask.Response:
+    try:
+        pid = int(PID_FILEPATH.read_text().strip())
+    except (OSError, ValueError):
+        return flask.Response("failed to read pidfile", status=500)
+
+    try:
+        os.kill(pid, 0)
+    except OSError:
+        return flask.Response(f"pid {pid} not running", status=500)
+    return flask.Response(f"ok: pid {pid} running", status=200)
```

Comment:
I had to look up why we were trying to kill the process here. :sweat_smile:  It seems sending `0` as the signal "will raise `OSError` if the process is dead, and return `None` if it is running". A comment or well-named function here would be great so it's clearer what we're trying to do.